haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="http://www.livejournal.com/~alivejournal/"
```

Comment:
Although this program is a fork of another program, you should use the fork's homepage:
```diff
-HOMEPAGE="http://www.livejournal.com/~alivejournal/"
+HOMEPAGE="https://github.com/grahams/alivejournal"
```
BTW, the fork's README already mentions the original's homepage.